File: code/ad188.pas
program Bai188;
var a, b: integer;
  function UCLN(m,n: integer): integer;
  begin
    while m <> n  do if m > n then m:= m - n else n := n - m;
    UCLN := m;
  end;
    
  begin
  readln(a, b);
  writeln((a*b) div UCLN(a,b));
  end.

File: code/ad188.cpp
#include <iostream>
using namespace std;

int ucln(int a,int b){
    while (a!=b){
        if (a>b)
            a=a-b;
        else
            b=b-a;
    }
}

int main() 
{
    int a,b,c;
    cin>>a>>b;
    c=ucln(a,b);
    c=a*b/c;
    cout<<c;
    return 0;
}